tests/10-issue.js

```diff
@@ -324,28 +430,34 @@ describe('Issuers - BitstringStatusList',
             'values.',
           async function() {
             this.test.link = 'https://www.w3.org/TR/vc-bitstring-status-list/#:~:text=The%20encodedList%20property%20of%20the%20credential%20subject%20MUST%20be%20a%20Multibase%2Dencoded%20base64url%20(with%20no%20padding)%20%5BRFC4648%5D%20representation%20of%20the%20GZIP%2Dcompressed%20%5BRFC1952%5D%20bitstring%20values%20for%20the%20associated%20range%20of%20verifiable%20credential%20status%20values.';
-            const {slc: {credentialSubject}} = await getSlc({issuedVc});
-            const {encodedList} = credentialSubject;
-            // Uncompress encodedList
-            const decoded = await sl.decodeList({encodedList});
-            should.exist(decoded,
-              'Expected encodedList to be a Multibase-encoded base64url' +
-              'representation of a GZIP-compressed bitstring.');
+            for(statusListCredential of statusListCredentials) {
+              const credentialSubject =
+                statusListCredential.credentialSubject;
+              const {encodedList} = credentialSubject;
+              // Uncompress encodedList
+              const decoded = await sl.decodeList({encodedList});
```

Comment:
might want to do something like:

```js
let decoded;
let error;
try {
  decoded = await sl.decodedList({encodedList});
} catch(e) {
  error = e;
} 
should.not.exist(error)
```

You get the idea. while it would definitely be outside of the scope of this PR, we have existing assertions for multibase encoding.
```